### DISCUSSION TITLE
The auction system is broken and needs fixing!

### DISCUSSION THREAD

**Discussion by LucasAG95** [sunflower-land/sunflower-land#3368] [General]
I am a player who has been with Sunflower for 2 years, maybe even more. And as I see it, the game is losing its way!

The game used to have the goal of players conquering SFT, whether with resources or SFL. It was something achievable by simply playing, even if it took time. The auction was introduced into the game, and it made sense until you decided to include BlockBucks as part of the auction. What's the problem with that? In this type of auction, the person is forced to pay for the item.

It's not like an SFL auction where the person bets their SFLs and, if they don't get the item, recovers their investment. In the case of BlockBucks, the person simply gives the money to you if they don't get the item. This causes people to spend money they may not have to, even if most of it comes from whales. It also takes away opportunities from people who were saving SFL. "But they could sell the SFL and buy BB to take the risk," you might say, but then they lose everything if they don't win the auction.

Having an auction with BB is already a joke, but having an item available only for BB and not for SFL or having it for BB but not for the seasonal ticket is absurd. BB should not be the main currency for the best items; it should be SFL and seasonal tickets!

"I know we have to pay the bills," I've heard a dev say when questioned about something. The thing is, you want to be a game, but players can't earn items by playing; they can only buy them (and not just buying, but gambling, because they might not win the auction, and their money is lost, as you don't refund the value of BBs). I don't know if this falls into malicious gambling, but I think it's poorly done. If you want to keep the auction, make it with SFL and seasonal tickets.

If you need to raise funds, you should look for more players and not make only the existing ones continue paying the bill. A game needs players!! You've been making a lot of promises, like the buds in $$, which came with a promise of various things for the beginning of the year and for those who held as well. Start thinking about the players and how to bring in more!

And one more thing, make Sunflower what it was supposed to be, a game with web3 freedom.

It's likely that the translation might not capture the exact tone, but I hope it conveys the main points effectively.

**Comment by rasheed19871** [comment]
Respect To everyone no offense off course just throwing my thoughts, I think auction with sfl is way better then bb because it strengthen the token thus attracting more players plus the hardcore formers rewarded, I don't know maybe the team has better plan but cutting some item from sfl is the one thing I am totally against I like and believe In the team but item only on bb it's better to give them outside mint from the game. I thought nfts are for the players to play with.

**Reply by LucasAG95** [reply]
"In full agreement!!"
NFTs should be for those who are playing, even though it's possible to buy SFL too, at least it helps the game. Now, spending BB is just an expense that doesn't contribute anything to the game.
"Plus, the devs have to pay the bills," but let them pay the bills by bringing in more players, thus increasing the BB turnover."

**Comment by adamhannigan** [comment]
Thanks for your feedback @Lusk2 

We've taken on feedback from last season and reduced the amount of Block Buck Auctions in the game. Last season, 100% of items had a Block Buck Auction. This season roughly half of items have a Block Buck Auction.

Majority of Auctions require SFL and Tulips - items only available through gameplay. The reason you have seen so many Block Buck Auctions in the first 2 weeks of the season is because we didn't want to start the Tulip Auctions too early while players were still collecting them.

The Block Bucks are our primary project revenue (90%) which allows us to continue building these seasons and launching new content.

We hear your concerns about making more items through gameplay only and will take it on board with future Auction and Stella requirements 🙏

**Reply by LucasAG95** [reply]
I understand that you need the revenue, and I apologize; you have reduced the items that require BB compared to last season, but in the previous season, it was also possible to obtain them with SFL (which helps with tokens). At the beginning of this season, the item considered the strongest was not even available for tulips; I believe there was a lot of complaint, and that's why they changed it (which is great, they listened and solved the problem). However, the main items this season (100 supply) are all for BB and tulips, none for SFL. Basically, all items that will make players spend the most. I understand that you need to maintain the game, and I love this game, but you have to understand the perspective of small/medium players who are betting money that, if lost, is gone. I myself bet on the hammer because I really wanted it, and I only got it on the third auction. I sold my GA and still had to borrow money in the end because I lost the first and second auctions, and if I hadn't succeeded, I would have lost all my money and would be without the GA. When I saw that I got the hammer, it was more of a feeling of relief than happiness. I believe the way of the last season is better, but that's just my opinion. If you are doing what is best for the project, that's fine, but after this, I won't risk it anymore. It's too much money for an average player to lose.

**Reply by adamhannigan** [reply]
I can see the frustration - thanks for sharing your feedback.

We have heard this sentiment from multiple different players. We've made a few adjustments to the Auctions - next season we will ensure these are more balanced 🙏
